mcs/class/System/Test/System.Net.Security/SslStreamTest.cs

```diff
@@ -93,6 +171,167 @@ void AuthenticateClientAndServer (bool server, bool client)
 		}
 	}
 
+	[Category("NotWorking")]    // Works in .Net, doesn't work in mono.
+	[Test()]
+	public void SelfCompatibilityTest()
+	{
+		#if __MonoCS__
```

Comment:
See http://www.mono-project.com/docs/faq/technical/#how-to-detect-the-execution-platform